Pages/FeedbackPage/Feedback.cshtml

```diff
@@ -0,0 +1,119 @@
+                                    <input asp-for="Feedback.Points" type="range" value="1" min="1" max="10" oninput="this.nextElementSibling.value = this.value">
+                                    <output>1</output>
+                                </div>
+                                <div class="mb-3">
+                                    <input type="submit" value="Submit" class="btn btn-dark" />
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+                <div class="tab-pane fade" id="restaurant">
+                    <div class="row border rounded shadow-sm">
+                        <div class="col">
+                            <form method="post">
+                                <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+                                <div class="mb-3">
+                                    <label asp-for="Feedback.Email" class="control-label"></label>
+                                    <input asp-for="Feedback.Email" class="form-control" />
+                                    <span asp-validation-for="Feedback.Email" class="text-danger"></span>
+                                </div>
+                                <div class="mb-3">
+                                    <label asp-for="Feedback.RestaurantName" class="control-label">Restaurant Name</label>
+                                    <input asp-for="Feedback.RestaurantName" class="form-control" />
+                                    <span asp-validation-for="Feedback.RestaurantName" class="text-danger"></span>
+                                </div>
+                                <div class="mb-3">
+                                    <label asp-for="Feedback.Answer" class="control-label"></label>
+                                    <textarea asp-for="Feedback.Answer" class="form-control" rows="3"></textarea>
+                                    <span asp-validation-for="Feedback.Answer" class="text-danger"></span>
+                                </div>
+                                <div class="mb-3">
+                                    <label asp-for="Feedback.Points">Points (between 1 and 10):</label>
+                                    <input asp-for="Feedback.Points" type="range" value="1" min="1" max="10" oninput="this.nextElementSibling.value = this.value">
+                                    <output>1</output>
+                                </div>
+                                <div class="mb-3">
+                                    <input type="submit" value="Submit" class="btn btn-dark" />
+                                </div>
+                            </form> 
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+            <!--<div class="row">
+        <div class="col-md-4">
```

Comment:
Is all of this commented section needed?

Reply:
Not anymore. Forgot to delete it but will do.